src/QuickAdd/Results.tsx

```diff
@@ -183,7 +183,7 @@ const Results: React.FC = () => {
   const resultsRef = createRef<HTMLUListElement>()
   const selectedRef = createRef<HTMLDivElement>()
 
-  const libraryContext = results[0].resultType === RESULTS_CONTEXTS.library
+  const libraryContext = results[0] && results[0].resultType === RESULTS_CONTEXTS.library
```

Comment:
Wild, I kind of would have expected typescript to tell us that this could be an empty array and that `results[0]` might be `undefined`.  Anyway FWIW we have access to the ["safe navigation" / "optional chaining" operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html), so in the future we might do `results[0]?.resultType` instead of the ol' tower of `&&` for nested access!